go.mod

```diff
@@ -34,6 +34,7 @@ require (
 	github.com/pingcap/kvproto v0.0.0-20200827082727-23dedec2339b
 	github.com/pingcap/log v0.0.0-20200511115504-543df19646ad
 	github.com/pingcap/sysutil v0.0.0-20200715082929-4c47bcac246a
+	github.com/pkg/errors v0.9.1
```

Comment:
Please use `github.com/pingcap/errors`, and add relevant information in `pkg/errs/errno` according to the needs of the log.